src/applicants/applicantsController.js

```diff
@@ -108,10 +111,43 @@ const changeApplicantStatus = async (req, res, next) => {
   }
 };
 
+const uploadApplicantImage = async (req, res, next) => {
+  const { id } = req.params;
+  try {
+    if (!(req.files && req.files[0] && req.files[0].path)) {
```

Comment:
do you thin we need here also a check like `req.files.length` ?

Reply:
You're right, I do need to check for length first.